m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
Actually, pbs_share.h is there exactly for this purpose.  It is a shared header file between daemons.